stdlib/3/multiprocessing/synchronize.pyi

```diff
@@ -0,0 +1,53 @@
+from typing import Callable, ContextManager, Optional, Union
+
+from multiprocessing.context import BaseContext
+import threading
+
+class Barrier(threading.Barrier):
+    def __init__(self,
+                 parties: int,
+                 action: Optional[Callable] = ...,
+                 timeout: Optional[float] = ...) -> None: ...
+
+class BoundedSemaphore(Semaphore):
+    def __init__(self, value: int = ..., *, ctx: BaseContext) -> None: ...
+
+_LockLike = Union[Lock, RLock]
+
+class Condition(ContextManager[bool]):
+    def __init__(self,
+                 lock: Optional[_LockLike] = ...,
+                 *,
+                 ctx: BaseContext) -> None: ...
+    def notify(self, n: int = ...) -> None: ...
```

Comment:
This argument doesn't exist in 3.6 (https://github.com/python/cpython/blob/3.6/Lib/multiprocessing/synchronize.py#L271) but does in master (https://github.com/python/cpython/blob/master/Lib/multiprocessing/synchronize.py#L271). We can probably add a version_info check for 3.7.